pysat/tests/test_instrument.py

```diff
@@ -45,9 +46,9 @@ def test_basic_instrument_load(self):
         data)."""
         self.testInst.load(2009, 1)
         test_date = self.testInst.index[0]
-        test_date = pysat.datetime(test_date.year, test_date.month,
-                                   test_date.day)
-        assert (test_date == pds.datetime(2009, 1, 1)) & \
+        test_date = dt.datetime(test_date.year, test_date.month,
+                                test_date.day)
+        assert (test_date == dt.datetime(2009, 1, 1)) & \
```

Comment:
flake8 will want the operator at the beginning of the next line.

Reply:
In other places, I've been rewriting these as separate assert statements.  Could do that here.